Library/Formula/xsd.rb

```diff
@@ -0,0 +1,49 @@
+class Xsd < Formula
+  desc "XML Data Binding for C++"
+  homepage "http://www.codesynthesis.com/products/xsd/"
+  url "http://www.codesynthesis.com/download/xsd/4.0/macosx/i686/xsd-4.0.0-i686-macosx.tar.bz2"
```

Comment:
Ok, I misunderstood you. You can make a pull-request to [Homebrew/homebrew-binary](http://github.com/Homebrew/homebrew-binary.